test/units/executor/test_task_executor.py

```diff
@@ -29,16 +30,19 @@
 from ansible.parsing.yaml.objects import AnsibleUnicode
 from ansible.utils.unsafe_proxy import AnsibleUnsafeText, AnsibleUnsafeBytes
 from ansible.module_utils.six import text_type
+from ansible.playbook.play_context import PlayContext
+from ansible.playbook.task import Task
 
 from collections import namedtuple
 from units.mock.loader import DictDataLoader
 
 
-get_with_context_result = namedtuple('get_with_context_result', ['object', 'plugin_load_context'])
+get_with_context_result = namedtuple(
+    "get_with_context_result", ["object", "plugin_load_context"]
+)
```

Comment:
don't include random formatting changes

Reply:
They're not random, I ran 'black' (see https://black.readthedocs.io/en/stable/) - now reverted